bindgen/src/program.ts

```diff
@@ -67,11 +68,13 @@ const specOption = program
   .argParser((arg, previous: ReadonlyArray<string> = []) => [...previous, parseExistingFilePath(arg)])
   .makeOptionMandatory();
 
+const optInOption = program
+  .createOption("--opt-in <opt-in list>", "Path of the 'opt-in list' specification")
```

Comment:
Should we document what happens if you leave out the opt-in?

Reply:
That is def helpful 🙂  I can add that in the code you've highlighted and in the description of this PR as well. If `--opt-in` is left out, all of the methods and fields will have `isOptedInTo = false`. So templates that choose to only produce bindings where that is `true` won't have any bindings generated.